app/code/Magento/Directory/Model/PriceCurrency.php

```diff
@@ -148,13 +146,18 @@ protected function getStore($scope = null)
     }
 
     /**
-     * Round price
-     *
-     * @param float $price
-     * @return float
+     * {@inheritdoc}
```

Comment:
For static tests
```suggestion
     * @inheritdoc
```

Reply:
Please note that there are 4266 instances of {@inheritdoc} in the 2.3 codebase, which is actually more then just @inheritdoc. So not really clear for me why we should need this change.

Reply:
Thanks! Static tests are running only on changed files. Also, afaik, a check for {@inheritdoc} was added recently.